lib/friends/introvert.rb

```diff
@@ -216,7 +220,11 @@ def add_nickname(name:, nickname:)
     # @param tag [String] the tag to add to the friend, of the form: "@tag"
     # @raise [FriendsError] if 0 or 2+ friends match the given name
     def add_tag(name:, tag:)
-      raise FriendsError, "Tag cannot be blank" if tag == "@"
+      if tag == "@"
```

Comment:
Same question here—do we need these conditionals to be nested or could we just do this?

```ruby
raise FriendsError, "Expected \"[Friend Name]\" \"[Tag]\"" if name.empty?
raise FriendsError, "Tag cannot be blank" if tag == "@"
```

Reply:
I'm thinking about this one as well as the point made to add_nickname🙂